lib/vrl/stdlib/benches/benches.rs

```diff
@@ -672,6 +673,35 @@ bench_function! {
     }
 }
 
+bench_function! {
+    is_empty => vrl_stdlib::IsEmpty;
+
+    empty_array {
+        args: func_args![value: value!([])],
+        want: Ok(true),
+    }
+
+    non_empty_array {
+        args: func_args![value: value!([1, 2, 3])],
+        want: Ok(true),
+    }
+
+    empty_object {
+        args: func_args![value: value!({})],
+        want: Ok(value!(true)),
```

Comment:
Can these be reduced to

```suggestion
        want: Ok(true),
```

as in the other assertions?